c10/core/TensorOptions.h

```diff
@@ -528,7 +537,7 @@ struct C10_API TensorOptions {
   // NB: We didn't use c10::optional here, because then we can't pack
   // the has_***_ boolean fields.
 
-  caffe2::TypeMeta dtype_ = caffe2::TypeMeta::Make<float>(); // 64-bit
+  caffe2::TypeMeta dtype_ = caffe2::TypeMeta::Make<float>(); // 16-bit
```

Comment:
Now that it's 16-bit it will be profitable to move this field down one, so it can get coalesced with layout and memory format.

Reply:
Yessss